ENVIRONMENTVARS.md

```diff
@@ -50,6 +50,7 @@ These environment variables control how certain elements on the page render and
 | `LIB_ANSWERS_EMAIL` | string | "email@email.com" | The email used in the `Feedback` component for the destination field. |
 | `loada11y` | boolean | true, false | Used to turn on the `react-a11y` package for accessibility testing. Only use in development mode. |
 | `LOGIN_URL` | string | "" | The URL to log a user into the NYPL Catalog. The user will be redirected to sign in with their NYPL credentials. This needs a valid value in order to sign in successfully, and it needs the "my-account" value in the `FEATURES` environment variable set. |
+| `LOGIN_BASE_URL` | string | "" | The base URL used for user authentication in the NYPL Catalog. The user will be redirected to sign in with their NYPL credentials. This needs a valid value in order to sign in successfully, and it needs the "my-account" value in the `FEATURES` environment variable set. |
```

Comment:
The description isn't exactly right. The first sentence is okay but afterwards, the variable won't be used just to redirect a user to sign in. This is also not dependent on the "my-account" feature. For example, this variable is used in the `LogoutLink` component but is not rendered conditionally with the "my-account" feature https://github.com/NYPL/discovery-front-end/blob/development/src/app/components/SubNav/SubNav.jsx#L63.
I think you can say that for now this variable is used to set the base authentication domain and used to get the appropriate logout link.

Reply:
Good call. Meant for it to encapsulate the LOGIN_URL description, but it makes sense to hold off til that env var is deprecated. Just updated the description.